src/tests/ftest/util/daos_utils.py

```diff
@@ -297,7 +297,28 @@ def pool_set_attr(self, pool, attr, value, sys_name=None):
 
         """
         return self._get_result(
-            ("pool", "set-attr"), pool=pool, attr=attr, value=value,
+            ("pool", "set-attr"), pool=pool, attr=':'.join([attr, value]),
+            sys_name=sys_name)
+
+    def pool_set_attrs(self, pool, attrs, sys_name=None):
+        """Set multiple pool attributes.
+
+        Args:
+            pool (str): Pool UUID.
+            attrs (map): Attribute name/value map.
```

Comment:
`map` in python has a different meaning
```suggestion
            attrs (dict): Attribute name/value dictionary.
```

Reply:
Oh, right. Thanks.